woocommerce/woocommerce-admin
# Onboarding - Add business extension recommendations

Fixes #2518.

This PR adds the Mailchimp and Facebook extensions to the business tab of the profile wizard.

<img width="849" alt="Screen Shot 2019-07-01 at 4 54 59 PM" src="https://user-images.githubusercontent.com/689165/60466647-3d227400-9c22-11e9-9046-54bc85cad664.png">

Note: I noticed certain values for `other_platform` are causing save endpoints to fail with a 400, even though they should be valid. This is not addressed in this PR. I can take a look at this tomorrow and fix in a follow-up PR.

### Detailed test instructions:

* Go to `/wp-admin/admin.php?page=wc-admin#/?step=business-details`
* Fill out the form and see that the extension recommendations display. Test the toggles, and saving your profile data. You can request `GET /wp-json/wc-admin/v1/onboarding/profile` to make sure the extensions saved.
